Adjustments to Badminton terminology

eol-account opened the pull request:
Replaced "set" with "game" and "leg" with "match"

eol-account committed:
Adjustments to Badminton terminology

Replaced "set" with "game" and "leg" with "match"

stephenrob commented on public/admin/templates/badminton.tmpl.html:
```diff
@@ -97,8 +97,8 @@
                     <div class="ui icon buttons">
-                        <button class="ui green button" ng-click="badminton.leg1 = badminton.leg1 + 1;"><i class="plus icon"></i></button>
-                        <button class="ui red button"  ng-click="badminton.leg1 = badminton.leg1 - 1" ng-disabled="badminton.leg1 == 0"><i class="minus icon"></i></button>
+                        <button class="ui green button" ng-click="badminton.match1 = badminton.match1 + 1;"><i class="plus icon"></i></button>
```
Display of the match score needs updating to reflect match over leg reference

stephenrob commented on public/admin/templates/badminton.tmpl.html:
```diff
@@ -123,8 +123,8 @@
                     <div class="ui icon buttons">
-                        <button class="ui green button" ng-click="badminton.leg2 = badminton.leg2 + 1;"><i class="plus icon"></i></button>
-                        <button class="ui red button"  ng-click="badminton.leg2 = badminton.leg2 - 1" ng-disabled="badminton.leg2 == 0"><i class="minus icon"></i></button>
+                        <button class="ui green button" ng-click="badminton.match2 = badminton.match2 + 1;"><i class="plus icon"></i></button>
```
Display of the match score needs updating to reflect match over leg reference

stephenrob reviewed (changes requested):
Couple of missed renames that need to be updated, comments are in relevant areas for these.

Text of `BEST OF 3 GAMES WINS MATCH` is incorrect under the score graphic - I think the wording of `BEST OF 3 MATCHES WINS` is correct based on the scoring system.

stephenrob commented on public/js/dashboard.js:
```diff
@@ -784,43 +784,5 @@ app.controller('badmintonCGController', ['$scope', 'socket',
             socket.emit("badminton:get");
         }
-
-        $scope.reset1 = function() {
```
Being able to reset scores would be useful as well

eol-account committed:
Corrections made to badminton scoring

Also added a reset button

stephenrob commented on public/js/dashboard.js:
```diff
@@ -773,6 +773,16 @@ app.controller('badmintonCGController', ['$scope', 'socket',
             $scope.badminton = msg;
         });
 
+        $scope.reset1 = function() {
```
If reset could only reset the current set of points (match score) and not the overall game score that would be better.

stephenrob commented on public/js/dashboard.js:
```diff
@@ -773,6 +773,16 @@ app.controller('badmintonCGController', ['$scope', 'socket',
+            $scope.badminton.game1 = 0;
+        };
+
+        $scope.reset2 = function() {
```
If reset could only reset the current set of points (match score) and not the overall game score that would be better.

stephenrob commented:
The admin updates are good, I may have slightly mislead you in some wording but asked someone who plays badminton to clarify the scoring and terminology for me.

Game: is correct wording for what it represents
Match: should be replaced by "points"

To win a game you have to have 21 points or above that be clear by 2 points.
3 games wins a match.

The text underneath should change to "Best of 3 games wins match

stephenrob commented on public/index.html:
```diff
@@ -231,18 +231,18 @@ <h2 ng-show="right.subHeading">{{right.subHeading}}</h2>
 
             <div class="row">
-                 <div class="footer">best of 3 games wins match</div>
+                 <div class="footer">best of 3 matches wins</div>
```
The original version of this was right - see general pull request comments where I explain why.

eol-account committed:
Changes to reset buttons and terminology

There's now individual reset buttons for each match and game score for both teams. Also changed "best of 3 matches wins" to "best of 3 games wins match"

Commit:
Merge branch 'master' into Badminton

stephenrob commented:
@thomasslee97 are you able to change the "match" column into "points" to reflect the correct terminology - all other changes are fine though - I do like the individual reset buttons.

The columns should be Game and Points, not Game and Match - this would need to be updated on both the admin and actual graphic.

eol-account committed:
Changed "Match" to "Points"

eol-account committed:
Merge remote-tracking branch 'origin/Badminton' into Badminton
